docs/agent-reference/how-to-run-agent.md

```diff
@@ -6,14 +6,32 @@
 AducIotAgent '<IoT device connection string>' [options...]
 ```
 
-### Additional Arguments
+e.g.
+```bash
+AducIotAgent '<IoT device connection string>' --enable-iothub-tracing --log-level 1
+```
 
-#### --EnableIotHubTracing
+### Options Details
 
-With this option enabled, the reference agent will output verbose tracing from
+`--version` tells the reference agent to output the version.
+
+`--enable-iothub-tracing` tells the reference agent to output verbose tracing from
 the Azure IoT C SDK. This option is useful for troubleshooting connection
 issues.
 
+`--health-check` tells the reference agent to turn on Health Check, which performs
+necessary checks to determine whether ADU Agent can function properly.  
+Currently, we are performing the following:  
+    - Implicitly check that agent process launched successfully.  
+    - Check that we can obtain the connection info.
```

Comment:
>we [](start = 17, length = 2)

as above (so below :-) )

Reply:
:smile: :grinning: 

---
In reply to: [612778372](https://github.com/Azure/iot-hub-device-update/pull/39#discussion_r612778372) [](ancestors = 612778372)